include/xsimd/config/xsimd_config.hpp

```diff
@@ -245,7 +245,7 @@
 #define XSIMD_WITH_SSE3 1
 #endif
 
-#if XSIMD_WITH_SSE3
+#if XSIMD_WITH_SSE3 || (defined(_M_IX86_FP) && _M_IX86_FP >= 2)
```

Comment:
Should it be `|| (!defined(__clang__) && (defined(_M_IX86_FP) && _M_IX86_FP >= 2))` ? Can you try with clang-cl ?

Reply:
Clang-cl supports the `_M_IX86_FP` flag correctly (https://github.com/microsoft/clang/blob/master/test/Driver/cl-x86-flags.c) so this PR stunt negatively impact that target